binaries/cli/src/main.rs

```diff
@@ -306,7 +319,7 @@ fn run() -> eyre::Result<()> {
                             let localhost = Ipv4Addr::new(127, 0, 0, 1);
                             (localhost, DORA_COORDINATOR_PORT_DEFAULT).into()
                         });
-                        Daemon::run(addr, machine_id.unwrap_or_default()).await
+                        Daemon::run(addr, machine_id.unwrap_or_default(), bind).await
```

Comment:
Similar thing, I think this refers to the `daemon_address` and `addr` refers to the coordinator addr. Could be good to have a better naming so that this is not confusing.